topology/University of Nebraska/Nebraska-Lincoln/CachingInfrastructure.yaml

```
@@ -169,3 +169,32 @@ Resources:
         Description: HCC StashCache Origin Server
     AllowedVOs:
       - HCC
+      
+    LIGO-StashCache-Origin:
```

@hzhu16 this field needs to be indented

```suggestion
  LIGO-StashCache-Origin:
```